src/collection.ts

```diff
@@ -95,4 +97,11 @@ export class Collection {
     }
     return this.pod.locales;
   }
+
+  /**
+   * Returns a list of documents in this collection (recursively).
+   */
+  docs() {
+    return this.pod.docs([`${this.path}**`]);
```

Comment:
Do we ensure that the path ends with a `/` when it is being set?

If we don't it would match files with the same name, correct?

ex:

```
/content/pages/sub/_collection.yaml
/content/pages/sub/page.yaml
/content/pages/subtract.yaml
```

`/content/pages/sub**` would match both `/content/pages/sub/page.yaml` and `/content/pages/subtract.yaml` if I understand glob correctly?